backend/tests/test_data/facilities.json

```diff
@@ -1,32 +1,27 @@
 [
-    {
-        "id":        2,
-        "created_at": "2024-06-27T19:15:00.123456789Z",
-        "updated_at": "2024-06-27T19:15:00.123456789Z",
-        "name":      "Jefferson City Correctional Center"
-    },
-    {
-        "id":        3,
-        "created_at": "2024-06-27T19:15:00.123456789Z",
-        "updated_at": "2024-06-27T19:15:00.123456789Z",
-        "name":      "Missouri Eastern Correctional Center"
-    },
-    {
-        "id":        4,
-        "created_at": "2024-06-27T19:15:00.123456789Z",
-        "updated_at": "2024-06-27T19:15:00.123456789Z",
-        "name":      "Potosi Correctional Center"
-    },
-    {
-        "id":        5,
-        "created_at": "2024-06-27T19:15:00.123456789Z",
-        "updated_at": "2024-06-27T19:15:00.123456789Z",
-        "name":      "Farmington Correctional Center"
-    },
-    {
-        "id":        6,
-        "created_at": "2024-06-27T19:15:00.123456789Z",
-        "updated_at": "2024-06-27T19:15:00.123456789Z",
-        "name":      "Algoa Correctional Center"
-    }
+  {
```

Comment:
removed the `id` fields of all the stored `.json` items, as not to screw up adding new items to a database that has been seeded. currently will throw a duplicate primary key error when you create a new program or user in a database that has been seeded with test data